src/main/java/Application.java

```diff
@@ -0,0 +1,49 @@
+import controller.GameController;
+import domain.Car;
+import domain.repository.CarRepository;
+import service.GameService;
+import service.validate.InputVerifier;
+import util.Converter;
+import view.input.InputView;
+import view.output.OutputView;
+
+import java.util.List;
+
+public class Application {
+    static GameService gameService = new GameService(new CarRepository());
+    static GameController gameController = new GameController(gameService);
+
+    public static void main(String[] args) {
+        String[] names = getNames();
+        List<Car> cars = makeCars(names);
+        int count = getCount();
+        OutputView.printPhrase();
+        execute(count, cars);
+        OutputView.printResult(cars);
+    }
+
+    private static String[] getNames() {
+        String inputNames = InputView.inputCarName();
+        String[] names = Converter.splitInput(inputNames);
+        InputVerifier.validateNameLength(names);
+        return names;
+    }
+
+    private static List<Car> makeCars(String[] names) {
+        return gameController.getCars(names);
+    }
+
+    private static int getCount() {
+        String inputCount = InputView.inputCount();
+        InputVerifier.checkInputTypeIsNumber(inputCount);
+        return Converter.convertType(inputCount);
+    }
+
+    private static void execute(int count, List<Car> cars) {
+        for (int step = 0; step < count; step++) {
+            gameController.updateStep(cars);
+            OutputView.printStep(cars);
+            System.out.println();
```

Comment:
step 변수가 for문 내에 사용되지 않고 있는것 같은데, 

```suggestion
        while (step-- > 0) {
            gameController.updateStep(cars);
            OutputView.printStep(cars);
            System.out.println();
```
등으로 사용할 수 있을 것 같아요.

Reply:
step이라는 변수를 만들지 않고 매개변수 count를 이용해 반복문을 작성할 수 있다는 생각을 하지 못했습니다.. 좋은 조언 감사합니다!